app/src/main/java/com/codepath/apps/critterfinder/fragments/SwipeablePetsFragment.java

```diff
@@ -60,4 +69,26 @@ private void setupPet() {
                 load("http://scontent.cdninstagram.com/t51.2885-15/s640x640/sh0.08/e35/12071062_736784449785114_1231793951_n.jpg").
                 into(mPetImage);
     }
+
+    // update the View with the image and data for a Pet
+    private void updateViewWithPet(PetModel petModel) {
+        this.mPetName.setText(petModel.getName());
+        this.mPetGender.setText(petModel.getSex());
+        Picasso.with(mPetImage.getContext()).load(petModel.getImageUrl()).into(mPetImage);
+    }
+
+    // start a pet search call
+    public void doPetSearch() {
+ //       loadingProgress.setVisibility(View.VISIBLE);
+        petSearch.doPetSearch();
+    }
+    public void onPetSearchSuccess(String result) {
```

Comment:
eventually when we wire up search we might want to pass the new list of pets (or pet in this case) in the onPetSearchSuccess callback.

i.e.
```
public void onPetSearchSuccess(List<PetModel> newPets)
```